datasets/imagenet2012/README.md

```diff
@@ -0,0 +1,1235 @@
+
+#### Who are the source language producers?
+
+WordNet synsets further quality controlled by human annotators. The images are from Flickr.
+
+### Annotations
+
+#### Annotation process
+
+The annotation process of collecting ImageNet for image classification task is a three step process.
+
+1. Defining the 1000 object categories for the image classification task. These categories have evolved over the years.
+1. Collecting the candidate image for these object categories using a search engine.
+1. Quality control on the candidate images by using human annotators on Amazon Mechanical Turk (AMT) to make sure the image has the synset it was collected for.
+
+See the section 3.1 in [1](https://arxiv.org/abs/1409.0575) for more details on data collection procedure and [2](https://ieeexplore.ieee.org/abstract/document/5206848) for general information on ImageNet.
+
+#### Who are the annotators?
+
+Images are automatically fetched from an image search engine based on the synsets and filtered using human annotators on Amazon Mechanical Turk. See [1](https://arxiv.org/abs/1409.0575) for more details.
+
+### Personal and Sensitive Information
+
+The 1,000 categories selected for this subset contain only 3 people categories (scuba diver, bridegroom, and baseball player) while the full ImageNet contains 2,832 people categories under the person subtree (accounting for roughly 8.3% of the total images). This subset does contain the images of people without their consent. Though, the study in [[1]](https://image-net.org/face-obfuscation/) on obfuscating faces of the people in the ImageNet 2012 subset shows that blurring people's faces causes a very minor decrease in accuracy (~0.6%) suggesting that privacy-aware models can be trained on ImageNet. On larger ImageNet, there has been [an attempt](https://arxiv.org/abs/1912.07726) at filtering and balancing the people subtree in the larger ImageNet.
+
+## Considerations for Using the Data
+
+### Social Impact of Dataset
+
+ImageNet dataset has been very crucial in advancement of deep learning technology as being the standard benchmark for vision models. The dataset aims to probe models on their understanding of the objects and has successfully shouldered the responsibility of holding through the test of time for that purpose. ImageNet is still one of the major datasets on which models are evaluated for their generalization in computer vision capabilities as the field moves towards self-supervised algorithms. Please see the future section in [1](https://arxiv.org/abs/1409.0575).
```

Comment:
"The ImageNet dataset"?

Reply:
"and has successfully shouldered the responsibility of holding through the test of time for that purpose." == "and has become the *de facto* dataset for this purpose?"

Not sure about successfully shouldering :)

Reply:
"Please see the future section in [1](https://arxiv.org/abs/1409.0575)." -- for what?